packages/node/src/schematics/library/library.ts

```diff
@@ -81,9 +83,12 @@ function createFiles(options: NormalizedSchema): Rule {
         offsetFromRoot: offsetFromRoot(options.projectRoot)
       }),
       move(options.projectRoot),
-      options.unitTestRunner === 'none'
-        ? filter(file => !file.endsWith('spec.ts'))
-        : noop()
+      options.unitTestRunner === 'jest'
```

Comment:
The last one will work when new test runners are added so I don't see a reason to change this.